R/utilities-writing-report.R

```diff
@@ -369,3 +369,25 @@ addMarkdownToc <- function(tocContent, fileName, logFolder = getwd()) {
   )
   return(invisible())
 }
+
+#' @title setSimulationDescriptor
+#' @description Set workflow simulation set descriptor
+#' @param workflow A `Workflow` object
+#' @param text Character describing simulation sets 
+setSimulationDescriptor <- function(workflow, text){
+  validateIsOfType(workflow, "Workflow")
+  validateIsString(text, nullAllowed = TRUE)
+  
+  # Allows NULL which is translated by ""
+  workflow$setSimulationDescriptor(text %||% "")
+  return(invisible())
+}
+
+#' @title getSimulationDescriptor
+#' @description Get workflow simulation set descriptor
+#' @param workflow A `Workflow` object
+#' @return character describing simulation sets 
+getSimulationDescriptor <- function(workflow){
+  validateIsOfType(workflow, "Workflow")
+  retrun(workflow$getSimulationDescriptor())

```

Comment:
`return `instead of `retrun`
seems there are no unit tests for the function so far 😉

Reply:
Good catch ! Thanks